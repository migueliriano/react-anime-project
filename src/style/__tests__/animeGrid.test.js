import React from 'react';
import renderer from 'react-test-renderer';
import getMuiTheme from 'material-ui/styles/getMuiTheme';
import toJson from 'enzyme-to-json';

import { shallow } from 'enzyme';

import {
  LoadMoreContainer,
  CircularProgressStyle,
  GridContainer,
} from '../animeGrid';

const muiTheme = getMuiTheme();

describe('animeGrid style components', () => {
  it('Should render a LoadMoreContainer Component', () => {
    const tree = renderer.create(<LoadMoreContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a CircularProgressStyle Component', () => {
    const tree = shallow(
      <CircularProgressStyle />,
      {
        context: { muiTheme },
      },
    );
    expect(toJson(tree)).toMatchSnapshot();
  });

  it('Should render a GridContainer Component', () => {
    const tree = renderer.create(<GridContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
