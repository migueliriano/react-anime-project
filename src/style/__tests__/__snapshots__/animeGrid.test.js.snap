// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`animeGrid style components Should render a CircularProgressStyle Component 1`] = `
.c0 {
  text-align: center;
}

<CircularProgress
  className="c0"
  max={100}
  min={0}
  mode="indeterminate"
  size={40}
  thickness={3.5}
  value={0}
/>
`;

exports[`animeGrid style components Should render a GridContainer Component 1`] = `
<div
  className=""
/>
`;

exports[`animeGrid style components Should render a LoadMoreContainer Component 1`] = `
.c0 {
  text-align: center;
}

<div
  className="c0"
/>
`;
